data_preparation.py

```diff
@@ -137,24 +137,35 @@ def prepare_data_in_dir(input_dir, output_dir, rep_length, overlapping_reads, lo
             split_fastq_file(fastq_file=merged_reads, output_dir=output_dir, cpu_count=cpu_count, max_memory=max_memory)
         else:
             raise Exception(f"When using merge_opposing !")
```

Comment:
please make this into a useful error message..

Reply:
fixed